pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -59,10 +59,11 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 	}
 
 	return &ReconcileWindowsMachineConfig{client: mgr.GetClient(),
-			scheme:       mgr.GetScheme(),
-			k8sclientset: clientset,
-			tracker:      vmTracker,
-			windowsVMs:   windowsVMs,
+			scheme:         mgr.GetScheme(),
+			k8sclientset:   clientset,
+			tracker:        vmTracker,
+			windowsVMs:     windowsVMs,
+			nodeConfigurer: nodeconfig.NewNodeConfig(clientset),
```

Comment:
I am not able to understand why you need to do this. I don't see the utility of having generic node configuration entities

Reply:
I think I explained this in the above message, let me know if this is a separate issue.